liquid-rust-common/src/config.rs

```diff
@@ -16,6 +45,7 @@ pub static CONFIG: SyncLazy<Config> = SyncLazy::new(|| {
             .set_default("log_dir", "./log/")?
             .set_default("dump_constraint", false)?
             .set_default("dump_checker_trace", false)?
+            .set_default("assert_terminator_behavior", "Assume")?
```

Comment:
shouldn't this be "assume"?

Reply:
I thought so too, but serde doesn't seem to use FromStr when deserializing, so it has to be uppercase to match the enum variant.